metrics/setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='lst-metrics',
+    description='LST-Bench Metrics and analysis',
+    url='https://github.com/microsoft/lst-bench',
+    license='Apache License, 2.0',
+    version='0.1',
+    packages=find_packages(),
+    install_requires=[
+        'pandas>=1.5.3',
```

Comment:
I'm not that familiar with Python conventions, but the project contains a list of required packages that were added by @poojanilangekar :
https://github.com/microsoft/lst-bench/blob/main/metrics/notebooks/requirements.txt
Shouldn't this list match that one (and have a single source of truth rather than two as well)?